app/views/markets/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2 class="forms">Edit a Market</h2>
+
+
+<%= form_for @market, method: :patch, url: market_path(@market) do |f| %>
+
+<%= f.label :'Market Name:' %>
+	<br>
+	<%= f.text_field :name, class: 'field' %>
+	<br>
+	<%= f.label :'Street:' %>
+	<br>
+	<%= f.text_field :street, class: 'field' %>
+	<br>
+	<%= f.label :'City:' %>
+	<br>
+	<%= f.text_field :city, class: 'field' %>
+	<br>
+	<%= f.label :'County:' %>
+	<br>
+	<%= f.text_field :county, class: 'field' %>
+	<br>
+	<%= f.label :'State:' %>
+	<br>
+	<%= f.text_field :state, class: 'field' %>
+	<br>
+	<%= f.label :'Zip Code:' %>
+	<br>
+	<%= f.text_field :zip, class: 'field' %>
+	<br>
+
+	<%= f.submit %>
+	<% end %>
```

Comment:
I think Tammy is making this comment too. I like that you're using the same form for new and edit, but best practice is to put the form in a partial, then rendering that same partial inside your edit.html.erb and new.html.erb. Have you done partials? You name them with an underscore, like "_form.html.erb", then render them like this ```<%= render "form" %>``` More info: http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials